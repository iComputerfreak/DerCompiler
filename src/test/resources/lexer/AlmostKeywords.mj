abstracta
asserta
booleana
breaka
bytea
casea
catcha
chara
classa
consta
continuea
defaulta
doublea
doa
elsea
enuma
extendsa
falsea
finallya
finala
floata
fora
gotoa
ifa
implementsa
importa
instanceofa
interfacea
inta
longa
nativea
newa
nulla
packagea
privatea
protecteda
publica
returna
shorta
statica
strictfpa
supera
switcha
synchronizeda
thisa
throwsa
throwa
transienta
truea
trya
voida
volatilea
whilea
