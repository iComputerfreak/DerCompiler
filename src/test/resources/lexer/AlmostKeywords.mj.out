identifier abstracta
identifier asserta
identifier booleana
identifier breaka
identifier bytea
identifier casea
identifier catcha
identifier chara
identifier classa
identifier consta
identifier continuea
identifier defaulta
identifier doublea
identifier doa
identifier elsea
identifier enuma
identifier extendsa
identifier falsea
identifier finallya
identifier finala
identifier floata
identifier fora
identifier gotoa
identifier ifa
identifier implementsa
identifier importa
identifier instanceofa
identifier interfacea
identifier inta
identifier longa
identifier nativea
identifier newa
identifier nulla
identifier packagea
identifier privatea
identifier protecteda
identifier publica
identifier returna
identifier shorta
identifier statica
identifier strictfpa
identifier supera
identifier switcha
identifier synchronizeda
identifier thisa
identifier throwsa
identifier throwa
identifier transienta
identifier truea
identifier trya
identifier voida
identifier volatilea
identifier whilea
EOF
